lua/modules/tools/config.lua

```diff
@@ -225,8 +225,14 @@ function config.wilder()
 		),
 	})
 
+	local match_hl = require("utils").hlToRgb("String", false)
```

Comment:
In this case, we no longer need the `utils` module. This line can be replaced as follows:
```suggestion
	local string_fg = vim.api.nvim_get_hl_by_name("String", true).foreground
	local match_hl = string_fg ~= nil and string.format("#%06x", string_fg) or "#ABE9B3"
```
Note that `String` is not one of the builtin hl groups. We need to set a fallback value for it.

Reply:
Ok, will do.